lib/msf/core/opt_base.rb

```diff
@@ -45,35 +69,34 @@ def initialize(in_name, attrs = [], aliases: [])
           raise("Invalid Regex #{regex_temp}: #{e}")
         end
       end
-      self.aliases = aliases
     end
 
     #
     # Returns true if this is a required option.
     #
     def required?
-      return required
```

Comment:
Do we have an official policy on this? I'm a fan of explicit return declarations when there's any question about what we may kick back, but also for visibility. I can't argue with this case, if the reader doesn't see the returns here they probably need to brush up on their Ruby, but we do have people coming on from other languages...

Reply:
Was just trying to be consistent. We had a mix before.